roles/nf-core/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+
+- name: install nf-core
+  local_action:
+    module: pip
+    virtualenv_command: /lupus/ngi/irma3/virtualenv-15.0.0/virtualenv.py
+    virtualenv: "{{ nf_core_env }}"
+    name: nf-core
+    state: present
+
+- name: Download NextFlow
+  local_action:
+    get_url url="{{ nextflow_download_url }}" dest="{{ nf_core_env }}/bin" mode="u+rwx,g=rwx"
+
+- name: Install NextFlow
+  local_action:
+    module: command
+    _raw_params: "{{ nf_core_env }}/bin/nextflow"
+  environment:
+    NXF_LAUNCHER: "/scratch"
+    NXF_HOME: "{{ nf_core_env }}/nextflow"
+    NXF_WORK: "/scratch"
+
+- name: nf-core
+  command: "{{ nf_core_env }}/bin/nf-core -v download {{ item.name }} --singularity --compress none --outdir {{ sw_path }}{{ item.name }} --release {{ item.release }}"
+  environment:
+    PATH: "{{ nf_core_env }}/bin:{{ ansible_env.PATH }}"
+  args:
+    chdir: "{{ sw_path }}"
+  with_items: "{{ pipelines }}"
+
+- name: Create directories for singularity images
+  file:
+    path: "{{ ngi_containers }}/{{ item.name }}"
+    state: directory
+    mode: g+s
+  with_items: "{{ pipelines }}"
+
+- name: Move singularity images
+  shell: "mv {{ sw_path }}{{ item.name }}/singularity-images/nf-core-{{ item.name }}-{{ item.release }}.simg {{ ngi_containers }}/{{ item.name }}/"
+  with_items:
+    - "{{ pipelines }}"
```

Comment:
Didn't we say we were going to remove this, and leave the `nf-core download` dir structure?

Reply:
I've created #336 for it as its dependent on some pending PRs in nf-core.